README.md

```diff
@@ -148,23 +148,21 @@ To run a functional test cluster,
 
 	cd build
 	./path/to/ceph/src/vstart.sh -d -n -x -l
-	./bin/ceph -s
+	./bin/ceph -c ceph.conf -s
```

Comment:
I think it defaults to searching in /etc/ceph for the ceph.conf. When I ran just the ceph command (without specifying the ceph.conf in $PWD) it hung.